beacon-chain/core/blocks/payload_test.go

```diff
@@ -12,7 +12,7 @@ import (
 	"github.com/prysmaticlabs/prysm/testing/util"
 )
 
-func Test_MergeComplete(t *testing.T) {
+func Test_BellatrixComplete(t *testing.T) {
```

Comment:
This refers to the action of merge

Reply:
Thanks @terencechain , this one slipped :-)